Documentation/network/clustermesh/clustermesh.rst

```diff
@@ -183,13 +183,14 @@ clusters.
 
 .. note::
 
-   You can additionally opt in to :ref:`kvstoremesh` when enabling
-   Cluster Mesh. Make sure to configure the Cilium CLI in *helm* mode and run:
+   Starting from v1.16 KVStoreMesh is enabled by default.
+   You can opt out of :ref:`kvstoremesh` when enabling
+   Cluster Mesh. Make sure to configure the Cilium CLI in *helm* mode and run instead:
```

Comment:
Newer versions of the Cilium CLI dropped the support for the classic mode, so the explicit *helm* mode reference could be likely removed.

Reply:
Good point, fixed.